gulpfile.js

```diff
@@ -21,7 +21,8 @@ var fs = require("graceful-fs"),
     util = require('gulp-util'),
     postcss = require('gulp-postcss'),
     rtl = require('postcss-rtl'),
-    babel = require('gulp-babel');
+    babel = require('gulp-babel'),
+    sort = require('gulp-sort');
```

Comment:
`gulp-sort` is unmaintained, with the last release 8 years ago. This is why I went with manual storing under https://github.com/OrchardCMS/OrchardCore/pull/15735 (and we should have a better long-term solution anyway: https://github.com/OrchardCMS/OrchardCore/issues/15740).

Reply:
Yeah sorting them manually will work too. The trick is to sort the files after they are resolved.